src/header/header.scss

```diff
@@ -314,18 +314,24 @@
 }
 
 @mixin iui-header-split-button {
```

Comment:
Ellipsis is not working.
![image](https://user-images.githubusercontent.com/36186912/145535509-c654ce50-a351-482c-8fef-f79924d1bf38.png)

Reply:
What the hell, before building I pulled latest changes, it showed that my sources are in sync and it had this image with broken ellipsis, then pulled again and it pulled newer changes with everything working... And I did everything this morning (in Lithuania time morning)...